Sources/App/Scenes/SceneManager.swift

```diff
@@ -57,7 +57,7 @@ class SceneManager {
             }
         }
 
-        func resolve<T>(with possible: T) {
```

Comment:
Was this done by SwiftLint/swiftformat?

Reply:
Yes, it was done by swiftformat. I can create a rule that does not do this change.